examples/demo_actor/README.md

```diff
@@ -17,16 +17,10 @@ This document describes how to create an Actor(DemoActor) and invoke its methods
 
 You can install dapr SDK package using pip command:
 
-<!-- STEP 
```

Comment:
Applying the same fixes in `demo_actor` as I assume it would cause the same problem if the `dapr-ext-fastapi` server version overrides the local package installed by tox....